src/plugins/dashboard/server/usage/dashboard_telemetry_collection_task.ts

```diff
@@ -48,7 +48,7 @@ export function initializeDashboardTelemetryTask(
 
 export function scheduleDashboardTelemetry(logger: Logger, taskManager?: TaskManagerStartContract) {
   if (taskManager) {
```

Comment:
Can we make `taskManager` a required param so we can remove the `!` when using `scheduleDashboardTelemetry` in `plugin.ts`? We're checking for `if(plugins.taskManager)` there anyway?

Reply:
done